Jenkinsfile

```diff
@@ -108,231 +114,388 @@ pipeline {
+        * ----------
+        * Builds Zowe CLI and SDK bundles for both LTS and Next releases
+        ************************************************************************/
+        stage('Create Bundles') {
             when {
                 allOf {
                     expression {
                         return BRANCH_NAME.equals(MASTER_BRANCH)
                     }
                 }
             }
-            steps {
-                timeout(time: 10, unit: 'MINUTES') {
-
-                    sh "npm set registry https://registry.npmjs.org/"
-                    sh "npm set @zowe:registry ${ZOWE_ARTIFACTORY_URL}"
-                    withCredentials([usernamePassword(credentialsId: ARTIFACTORY_CREDENTIALS_ID, usernameVariable: 'USERNAME', passwordVariable: 'PASSWORD')]) {
-                        // TODO: Consider using tooling like artifactory-download-spec to get license.zip. Post-Infrastructure migration answer.
-                        sh "mkdir -p licenses && cd licenses && curl -fs -o zowe_licenses_full.zip $ZOWE_LICENSE_ZIP_URL"
-                        sh "./scripts/npm_login.sh $USERNAME $PASSWORD \"$ARTIFACTORY_EMAIL\" '--registry=${ZOWE_ARTIFACTORY_URL} --scope=@zowe'"
-                    }
-                    sh "npm install jsonfile"
-
-                    sh "npm pack @zowe/db2-for-zowe-cli@4.1.0"
-                    sh "npm pack @zowe/cics-for-zowe-cli@4.0.2"
-                    sh "npm pack @zowe/ims-for-zowe-cli@2.0.1"
-                    sh "npm pack @zowe/mq-for-zowe-cli@2.0.1"
-                    sh "npm pack @zowe/zos-ftp-for-zowe-cli@1.6.0"
-                    sh "./scripts/repackage_bundle.sh *.tgz"
-                    sh "mv zowe-cli-package.zip zowe-cli-plugins-${ZOWE_CLI_BUNDLE_VERSION}.zip"
-
-                    archiveArtifacts artifacts: "zowe-cli-plugins-${ZOWE_CLI_BUNDLE_VERSION}.zip"
-
-                    // Remove all tgzs after bundle is archived
-                    sh "rm -f *.tgz"
-                }
-            }
-        }
-        /************************************************************************
-         * STAGE
-         * -----
-         * Build Zowe NodeJS SDK Bundle
-         *
-         * TIMEOUT
-         * -------
-         * 10 Minutes
-         *
-         * EXECUTION CONDITIONS
-         * --------------------
-         * - Always
-         *
-         * DECRIPTION
-         * ----------
-         * Gets the latest version of the Zowe NodeJS SDK package from NPM
-         * Creates an archive with 'fat' versions of the Plugins -
-         *   dependencies are bundled.
-         *
-         * OUTPUTS
-         * -------
-         * A Zowe NodeJS SDK Archive.
-         ************************************************************************/
-        stage('Create Zowe NodeJS SDK Bundle') {
-            when {
-                allOf {
-                    expression {
-                        return BRANCH_NAME.equals(MASTER_BRANCH)
+            failFast true

```

Comment:
If we are doing a Dry Run, do we want to abort it on of the bundles fails?

Reply:
Are you suggesting to make `failFast` conditional based on whether DryRun is enabled?

Reply:
that'd be a cool idea, (not sure if it's possible though) 
I was just wondering if we would at least want the pipeline to keep going if it's a dry run 😋

Reply:
Agree this would be a cool idea, unfortunately it doesn't seem like it's supported 😢 
https://issues.jenkins.io/browse/JENKINS-52158

Is it ok to leave it true, or would you prefer that failFast = false? 😛